ci/test/00_setup_env_arm_mac.sh

```diff
@@ -0,0 +1,17 @@
+#!/usr/bin/env bash
```

Comment:
You may be interseted in this backport also: 
    Merge bitcoin/bitcoin#26388: ci: Use `macos-ventura-xcode:14.1` image for "macOS native" task

```
-export HOST=x86_64-apple-darwin
+export HOST=arm64-apple-darwin
```

```
git show bd478890c513aafe2c65d4199c9f3c558670eaa3:ci/test/00_setup_env_mac_native_arm64.sh
```